pkg/command/config.go

```diff
@@ -67,21 +69,56 @@ var getConfigCmd = &cobra.Command{
 	Use:               "get",
 	Short:             "Get the current configuration",
 	ValidArgsFunction: noMoreCompletions,
+	Args:              cobra.NoArgs,
 	RunE: func(cmd *cobra.Command, args []string) error {
 		cfg, err := configlib.GetClientConfig()
 		if err != nil {
 			return err
 		}
 
+		// Print the entire config
 		b, err := yaml.Marshal(&cfg)
 		if err != nil {
 			return err
 		}
-		fmt.Println(string(b))
+		fmt.Fprintln(cmd.OutOrStdout(), strings.TrimSpace(string(b)))
```

Comment:
This remove the extra empty line at the end of the output.
It also prints to `OutOrStdout()` so that unit tests can redirect the output